python/core/qgssettings.sip

```diff
@@ -156,6 +156,11 @@ Returns a list of all top-level keys that can be read using the QSettings object
 Returns a list of all key top-level groups that contain keys that can be read using the QSettings object.
  :rtype: list of str
 %End
+    QStringList globalChildGroups() const;
```

Comment:
Would you mind adding a test case for `globalChildGroups()` to  https://github.com/qgis/QGIS/blob/master/tests/src/python/test_qgssettings.py ?

Reply:
Thanks @elpaso. I discovered another minor issue writing the test. The test now covers the `settings.globalChildGroups()`.